### ISSUE TITLE
Member expression indentation rule

### ISSUE THREAD

**Issue by miguelangelalvarez** [eslint/eslint#6826]
**What version are you using?**

Version: 3.2.2

**What did you do?**

I’m using the indentation rule and after eslint 3.1.1 a new member expression rule has been introduced.

Now, this is what I’ve got:

```js
fetch(url, object)
.then((response) => { // Indentation error
  console.log(response);
})
```

**What happened?**

And this is what eslint expects:

```js
fetch(url, object)
  .then((response) => { // The error has disappeared
    console.log(response);
  })
```

**What did you expect to happen?**

I would like to deactivate the MemberExpression indentation rule because sometimes I don’t use indentation for member expressions and sometimes I do. I’d like to have the behaviour I had in 3.1.1.

The example above is one where I’m not using that indentation, but I use to use it when I’ve got code like the next:

```js
expect(object)
  .toBe(clonedObject)
```

**Comment by vitorbal** [comment]
Hmm, as far as I understood from the discussions, MemberExpression indentation checks should be off by default now.
@miguelangelalvarez could you also include the configuration used and the actual raw output from eslint when you run it?

**Comment by eslintbot** [comment]
Hi @miguelangelalvarez, thanks for the issue. It looks like there's not enough information for us to know how to help you. If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by miguelangelalvarez** [comment]
Hi @vitorbal, this is the rule we are using at this moment:

```js
"indent": [ 2, 2, { "SwitchCase": 1 } ]
```

As you can see, by default, the member expression rule is activated because we are using the indent rule.

We want to deactivate only the member expression rule.

**Comment by vitorbal** [comment]
@miguelangelalvarez I'm trying to reproduce this and I am unable to. The `MemberExpression` option of the `indent` rule is off by default, so I don't see why you would be getting an error for that with the rule turned on like you describe.
Could you please post the **full** configuration file and the **raw** output with the error that you get from eslint?


For reference, here's the full configuration file and test file that I used and was unable to reproduce your problem:
```json
{
    "root": true,
    "parserOptions": {
        "ecmaVersion": 6
    },
    "rules": {
        "indent": [ 2, 2, { "SwitchCase": 1 } ]
    }
}
```

```javascript
fetch(url, object)
.then((response) => {
  console.log(response);
});

fetch(url, object)
  .then((response) => {
    console.log(response);
  });
```

```shell
$ node_modules/.bin/eslint --version
v3.2.2

$ node_modules/.bin/eslint src/test.js
# no errors
```

**Comment by miguelangelalvarez** [comment]
Thanks again @vitorbal. You're right and I was wrong: the latest version fixes it and I was working with v3.2.0.

This commit has been the fix: https://github.com/eslint/eslint/commit/c7488ac6fe47d31f57fbdf72f3ef14fc4979486f

Will close the ticket.

**Comment by vitorbal** [comment]
@miguelangelalvarez thanks for the update, glad to hear it!
